sdk/translation/ai-translation-text-rest/src/index.ts

```diff
@@ -1,16 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import TextTranslation from "./customClient";
+import TextTranslationClient from "./textTranslationClient";
```

Comment:
We want to use customClient which contains our custom code. Question here is - is this correct way? Using fully custom file? Or is there any mechanism like _patch files in Python?